samza-api/src/main/java/org/apache/samza/operators/descriptors/base/stream/InputDescriptor.java

```diff
@@ -52,6 +57,7 @@
   private Optional<Boolean> isBootstrapOptional = Optional.empty();
   private Optional<Boolean> isBoundedOptional = Optional.empty();
   private Optional<Boolean> deleteCommittedMessagesOptional = Optional.empty();
+  private Optional<String> broadcastPartitionsOptional = Optional.empty();
```

Comment:
It would be nice to have a method isBroadcast() to indicate whether this InputDescriptor is broadcast stream or not.